juan-silva/Final-Project/project/src/pages/Games.js

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+import './Games.css';
+import who from '../assets/who.png'
+const url='https://pokeapi.co/api/v2/pokemon-form/';
+
+function Games(data){
+    
+    const [random,setRandom]=useState('');
+    const [picture,setPicture]=useState('');
+    const getRandomPokemon=()=>{
```

Comment:
@jsilva97 , Since you are not using Redux, all the logic and API consumption should be in a context